corehq/apps/analytics/templates/analytics/initial/gtm.html

```diff
@@ -0,0 +1,13 @@
+{% load hq_shared_tags %}
+{% initial_analytics_data 'gtm.apiId' ANALYTICS_IDS.GTM_ID %}
+{% if request.couch_user %}
+  {% initial_analytics_data 'gtm.userId' request.couch_user.userID %}
+  {% initial_analytics_data 'gtm.userIsDimagi' request.couch_user.is_dimagi %}
+  {% initial_analytics_data 'gtm.userIsCommCare' request.couch_user.is_commcare_user %}
```

Comment:
nit: I'd suggest saving that as `gtm.userIsCommCareUser` since that makes it more clear.

Reply:
Addressed in ac89b8aba537dd094044eaf1b2880d2c9b7318f2